client/sites-dashboard-v2/sites-dataviews/index.tsx

```diff
@@ -26,6 +27,15 @@ type Props = {
 	setDataViewsState: ( callback: ( prevState: DataViewsState ) => DataViewsState ) => void;
 };
 
+export const statuses = [
+	{ value: 1, label: __( 'All sites' ), slug: 'all' },
+	{ value: 2, label: __( 'Public' ), slug: 'public' },
+	{ value: 3, label: __( 'Private' ), slug: 'private' },
+	{ value: 4, label: __( 'Coming soon' ), slug: 'coming-soon' },
+	{ value: 5, label: __( 'Redirect' ), slug: 'redirect' },
+	{ value: 6, label: __( 'Deleted' ), slug: 'deleted' },
+];
```

Comment:
I moved these out of the component to make them an accessible export and tie them to the slugs we use for filtering on these values. This gives us a more direct connection between integer values used by the data component and our slugs for filtering.

Reply:
Great. At A4A, we created a map to target those integer values:

`client/a8c-for-agencies/sections/sites/constants.ts`

```
export const filtersMap: AgencyDashboardFilterMap[] = [
	{ filterType: 'all_issues', ref: 1 },
	{ filterType: 'backup_failed', ref: 2 },
	{ filterType: 'backup_warning', ref: 3 },
	{ filterType: 'threats_found', ref: 4 },
	{ filterType: 'site_disconnected', ref: 5 },
	{ filterType: 'site_down', ref: 6 },
	{ filterType: 'plugin_updates', ref: 7 },
];
```

And then we do things like this:

```
return {
	field: 'status',
	operator: 'in',
	value: filtersMap.find( ( filterMap ) => filterMap.filterType === issueType )?.ref || 1,
};
```

But we are still improving things there.